pagopa/io-app
# feat(Firma con IO): [SFEQS-1894] Add signature field static info tooltip with the definition of unfair clause

## Short description
This PR adds a new static bottom sheet with the definition of unfair clause. This could be a temporary solution waiting a new UX.

https://github.com/pagopa/io-app/assets/49342144/a1c52a08-dc8f-42fc-93ae-ee61b7a1a396

## List of changes proposed in this pull request
- Added new BS
- Update `FciSignatureFieldsScreen`
- Add new locales

## How to test
Using `io-dev-api-server` start a signing flow. On the `FciSignatureFieldsScreen` the UNFAIR clause should have an info tooltip to the right of the section header. Tap on the info icon to open the BS with the description.
